.github/workflows/release.yml

```diff
@@ -51,6 +51,12 @@ jobs:
           print("OK")
           EOF
 
+      - uses: actions/upload-artifact@v2
+        with:
+          name: jupyterhub-${{ github.sha }}
+          path: "dist/*"
+          if-no-files-found: error
+
```

Comment:
```suggestion
      # ref: https://github.com/actions/upload-artifact#readme
      - uses: actions/upload-artifact@v2
        with:
          name: jupyterhub-${{ github.sha }}
          path: "dist/*"
          if-no-files-found: error

```

Reply:
I find it generally useful to have a quick reference to external actions to find changelogs, inputs / outputs, etc.

Reply:
Woops the preview wasn't so nice now that I commented on multiple lines, won't do that again. I only added the reference link though.